images_config/README_images.md

```diff
@@ -0,0 +1,395 @@
+
+![alt text](doc/kickstart-0.png "Logo Title Text 0")
+Very basic and self explanatory settings here. We have used x86 architecture, because our devices had less than 4 GB of RAM. 
+
+
+![alt text](doc/kickstart-1.png "Logo Title Text 1")
+If you want to install Ubuntu from CD-ROM or USB like we did, choose CD-ROM. If you want to install it from ISO file stored on FTP, HTTP servers or hard drive, choose appropriate options. 
+
+
+![alt text](doc/kickstart-2.png "Logo Title Text 2")
+Keep boot loader options to default.
+
+
+![alt text](doc/kickstart-3.png "Logo Title Text 3")
+Be careful on this step and set the right partitioning information, because it can completely delete your current system. We have installed Ubuntu on machines that had the same size HDDs with existing partitions. 
+
+Make sure to create /boot, / ,and swap partitions. In this example the first two partitions are in fixed size and the last one is set to fill all remaining space for swap .
+
+Note:  We make some small changes in the partition information in the further settings of kick-start file.
+
+
+![alt text](doc/kickstart-4.png "Logo Title Text 4")
+Choose Static or DHCP. 
+
+
+![alt text](doc/kickstart-5.png "Logo Title Text 5")
+Kept the default settings. 
+
+
+![alt text](doc/kickstart-6.png "Logo Title Text 6")
+Enter your credentials. You can later change the password in ks.cfg file manually. If you chose to encrypt your password, the supported hash in Kickstart configuration is MD5. Use Open SSL command
+```
+openssl passwd -1 yourpassword 
+```
+in Terminal to generate the new password. Place the generated new password in the place of Password and Confirm Password.
+
+![alt text](doc/kickstart-7.png "Logo Title Text 7")
+Keep it disabled. Ubuntu doesn't support firewall settings. 
+
+
+![alt text](doc/kickstart-8.png "Logo Title Text 8")
+Do not configure the X Window System here. Ubuntu automaticlly solves this one anyway.
+
+
+![alt text](doc/kickstart-9.png "Logo Title Text 9")
+Ubuntu doesn't support Kickstart package selection. We will add them manually %packages section in ks.cfg file.
+
+![alt text](doc/kickstart-10.png "Logo Title Text 10")
+We have some specific per-installation script, we will add manually in %pre section in ks.cfg file.
+
+![alt text](doc/kickstart-11.png "Logo Title Text 11")
+We have some specific post-installation script, we will add manually in %post section in ks.cfg file.
+
+
+Write anything that you need to do after kickstart installation. It executes the script in chroot environment, so you don't need to use sudo.
+
+When you are finished with the configuration, press File > Save File in the top menu. Keep default name as ks.cfg and save it to your Desktop.
+
+In ks.cfg search for #Disk partitioning information where we are going make small changes in the partition according to our requirement. Remove the lines of partition and paste below mentioned lines.
+```
+#Disk partitioning information 
```

Comment:
please don't copy the code, just use a link to the lines:
https://github.com/ipa320/setup_cob4/blob/master/images_config/ks-robot-master.cfg#L33-L46